### ISSUE TITLE
How to use x flag on ios ?

### ISSUE THREAD

**Issue by heinekenpsx** [Al4ise/Azule#113]
Title :)
"iOS Only:
  -x [Apple ID] [Password]     Fetch and decrypt iPA with specified Apple ID"

**Comment by Al4ise** [comment]
azule -x [appleid] [password for appleid] -i [bundle id of app you want to decrypt] -o [output destination for decrypted ipa]

**Comment by heinekenpsx** [comment]
Thanks

**Comment by heinekenpsx** [comment]
@Al4ise azule only supports apple id in the US region?
I also can't use --country flag
I want to use apple id in Vietnam region, it gives invalid error

**Comment by Al4ise** [comment]
just pushed an update. -z can now be used to set a country code

**Comment by heinekenpsx** [comment]
Thanks

**Comment by heinekenpsx** [comment]
Hmm, invalid app
Did I not do it right?
xXx:~ mobile$ azule -x mail password -z AU -i com.swm.live -o ~
[*] Checking for App Updates...
[*] Invalid App

**Comment by Al4ise** [comment]
uh syntax looks fine.. does it do that with every app?

**Comment by heinekenpsx** [comment]
Yes, every app, i tried authenticating appld id with ipatool tool (by Majd Alfhaily) and it worked fine
xXx:~ root# ipatool search 7plus -c au
==> ℹ️  [Info] Searching for '7plus' using the 'au' store front...
==> ℹ️  [Info] Found 5 results:
1. 7plus: com.swm.live (4.33).
2. 9Now: au.com.ninemsn.jump-in (4.12.16).
3. SBS On Demand: au.com.sbs.ondemand (4.6.0).
4. Amazon Prime Video: com.amazon.aiv.AIVApp (9.17.1).
5. ABC iview: au.net.abc.ABCiView (5.1.2).
xXx:~ root# azule -x mail pwd -x AU -i au.net.abc.ABCiView -o ~
 -j
[*] Checking for App Updates...
[*] Invalid App
xXx:~ root# ipatool auth login
==> ⚠️  [Warning] Enter Apple ID email:
==> ⚠️  [Warning] Enter Apple ID password:
==> ℹ️  [Info] Authenticating with the App Store...
==> ℹ️  [Info] Authenticated as 'Khang Nguyen'.

**Comment by Al4ise** [comment]
hm, i have an idea, though im on vacation rn so im not sure when i'll implement it

**Comment by heinekenpsx** [comment]
> hm, i have an idea, though im on vacation rn so im not sure when i'll implement it

Ah, never mind, just do it when you feel free, thanks
